eth/vm/rstack.py

```diff
@@ -0,0 +1,68 @@
+from typing import (
+    List,
+    Tuple,
+    Union,
+)
+
+from eth.exceptions import (
+    InsufficientStack,
+    FullStack,
+)
+
+from eth.validation import (
+    validate_stack_int,
+)
+
+from eth_utils import (
+    big_endian_to_int,
+    ValidationError,
+)
+
+"""
+This module simply implements for the return stack the exact same design used for the data stack.
+As this stack must simply push_int or pop1_int any time a subroutine is accessed or left, only
+those two functions are provided.
+For the same reason, the class RStack doesn't inherit from the abc StackAPI, as it would require
+to implement all the abstract methods defined.
+"""
+
+
+class RStack:
```

Comment:
It's very rare that we use initials in method and class names. A friendlier name here IMO:
```suggestion
class ReturnStack:
```